### DISCUSSION TITLE
If the language is Japanese, Arithmetic components have the same label on property.

### DISCUSSION THREAD

**Discussion by kaedepon** [FakeFishGames/Barotrauma#13767] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Similar the Report at #13739
It is the property of arithmetic components.
In English, there are written as "Clamp max" and "Clamp min".
In Japanese, both are written as "最小値固定", which is confusing when using.

# In English
![component_en](https://github.com/FakeFishGames/Barotrauma/assets/47914478/8216979e-7a20-4e2d-a7f2-d5d0f4ba7920)

# In Japanese
![component_jp](https://github.com/FakeFishGames/Barotrauma/assets/47914478/9cf63e62-a00e-499a-8667-b53263b6a840)

I think it is typo this line in `JapaneseVanilla.xml`
`<sp.arithmeticcomponent.clampmax.name>最小値固定</sp.arithmeticcomponent.clampmax.name>`

In this case, I think Japanese will understand this translation.
"Clamp max" : "最大値" or "上限値"
"Clamp min" : "最小値" or "下限値"
*FYI) From a native's point of view, it feels more natural to remove "固定".
and it feels more meanings match to "上限値/下限値".

### Reproduction steps

1. Select Japanese Language.
2. Interact any arithmetic components (adder, subtract etc).
3. Watch the labels on this property.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! We've addressed this now at our end, the fix should be coming up in the next update.
